### DISCUSSION TITLE
GitHub Actions

### DISCUSSION THREAD

**Discussion by amyb12345** [docker/extension-ideas#49] [Ideas]
-  Test GitHub Actions locally using Docker

Note: This is a transfer from our initial method of collecting new extension requests. If you have additional information on how this will help you or other functionality you'd like to see, please add more details in the comments.

**Comment by ra312** [comment]
I would love to see how we can test different actions composed wit each other in Docker Extension for Github Actions

**Comment by mike-seagull** [comment]
This is already possible using `act`: https://github.com/nektos/act

**Comment by tippexs** [comment]
While working on my fine-tune for my DockerCon Talk and Workshop I came across this one and LOVE IT! As GH Actions are super need and as mentioned by `act` it could be an Ext. combining local testing and remote querying the status of some specific actions. I will chat with you @amyb12345 about this at DockerCon and update this thread!
